madmin/api/modules/ftr_device.py

```diff
@@ -280,6 +280,15 @@ class APIDevice(apiHandler.ResourceHandler):
                     "description": "Rotate accounts if player level >= 30 (for leveling mode)",
                     "expected": bool
                 }
+            },
+            "injection_thresh_reboot": {
+                 "settings": {
+                    "type": "text",
+                    "require": False,
+                    "empty": None,
+                    "description": "Reboot (if enable) device after not injecting for X times in a row (Default: 20)",

```

Comment:
```suggestion
                    "description": "Reboot (if enabled) device after not injecting for X times in a row (Default: 20)",
```

Reply:
@JabLuszko please approve this change to your PR. Will merge as soon as you've done so.